ddtrace/internal/encoding.py

```diff
@@ -48,17 +51,36 @@ def encode(self, obj):
         raise NotImplementedError()
 
 
-class JSONEncoder(_EncoderBase):
+class JSONEncoder(json.JSONEncoder, _EncoderBase):
     content_type = "application/json"
 
+    def __init__(self):
+        # Reduce unnecessary whitespace
+        super(JSONEncoder, self).__init__(separators=(",", ":"), indent=None)
```

Comment:
Unrelated? Should we do this separately in case some side-effect gets introduced here?

Reply:
removing whitespace or inheriting from standard `JSONEncoder`?

Reply:
the whitespace and separators (unless those are the defaults)

Reply:
removed, once this lands I'll open a follow-up PR to change.